@xen-orchestra/lite/src/views/settings/SettingsView.vue

```diff
@@ -124,21 +179,51 @@ import UiCard from "@/components/ui/UiCard.vue";
 import UiKeyValueList from "@/components/ui/UiKeyValueList.vue";
 import UiKeyValueRow from "@/components/ui/UiKeyValueRow.vue";
 
-const version = XO_LITE_VERSION;
-const gitHead = XO_LITE_GIT_HEAD;
+const xoLiteVersion = XO_LITE_VERSION;
+const xoLiteGitHead = XO_LITE_GIT_HEAD;
 const { locale } = useI18n();
 
+const poolStore = usePoolStore();
+const { pool, isReady } = storeToRefs(poolStore);
+const hostStore = useHostStore();
+
+const poolMaster = computed(() =>
+  pool.value ? hostStore.getRecord(pool.value?.master) : undefined
```

Comment:
Not really important, but since you test for `pool.value`, you can remove the `?`

```suggestion
  pool.value ? hostStore.getRecord(pool.value.master) : undefined
```